.travis.yml

```diff
@@ -12,45 +12,40 @@ before_cache:
 jobs:
   fast_finish: true
   include:
-    - name: Run Rust tests(linux)
-      language: rust
+    - name: Run tests (Linux)
+      language: python
```

Comment:
Have we ever tried using `language: rust` for these tests? Cause I think Python is already installed on the macos and linux environments, although it might not be the version we want.

Reply:
There is a comment: 
```
      # To test the snippets, we use Travis' Python environment (because
      # installing rust ourselves is a lot easier than installing Python)
```